### ISSUE TITLE
Question about Dataset Differences 

### ISSUE THREAD

**Issue by Cb1ock** [stoneMo/DeepAVFusion#4]
Hi Author, thanks for your great work !

I have a question regarding the dataset used in Table 4 of Section 4.2 of the paper. Specifically, could you clarify the differences between the VGGSound-All dataset and the original VGGSound dataset? I noticed the fine-tuning classification accuracy for CAV-MAE on VGGSound-All that you reported is 55.32, which is significantly lower than the 65.4 reported in the original CAV-MAE paper on VGGSound. Both evaluations used the base ViT architecture.

Even if some original data is no longer available for download and you reported your own reproduced results for fairness, such a significant performance drop is difficult to understand.

**Comment by Cb1ock** [comment]
@stoneMo

**Comment by stoneMo** [comment]
Hi, thanks for your interest in our work. The "VGGSound-All" dataset refers to the original VGGSound dataset. For the CAV-MAE, we reproduced the results on the original VGGSound dataset using the same fine-tuning hyperparameters.
